README.md

```diff
@@ -111,6 +109,20 @@ Use on-demand whole file annotations to see authorship, recent changes, and a he
 
 💡 On an active file, use the `Toggle File Blame`, `Toggle File Changes`, and `Toggle File Heatmap` commands from the Command Palette to turn the annotations on and off.
 
+## Commit Graph `Pro`
+
+Easily visualize your repository and keep track of all work in progress.
+
+Use the rich commit search to find exactly what you're looking for. Its powerful filters allow you to search by a specific commit, message, author, a changed file or files, or even a specific code change. [Learn more](https://gitkraken.com/solutions/commit-graph?utm_source=gitlens-extension&utm_medium=in-app-links)
+
+<figure align="center">
+  <img src="https://raw.githubusercontent.com/gitkraken/vscode-gitlens/main/images/docs/commit-graph.png" alt="Commit Graph" />
```

Comment:
@eatmorespinach not a deal-breaker for the release, but this image is outdated and doesn't include the new filter option that will show only commits in the graph that match the search query.